datahub-web-react/src/graphql/user.graphql

```diff
@@ -29,7 +29,7 @@ query getUser($urn: String!, $groupsCount: Int!) {
         }
         relationships(
             input: {
-                types: ["IsMemberOfGroup", "IsMemberOfNativeGroup"]
+                types: ["IsMemberOfGroup", "IsMemberOfNativeGroup", "IsMemberOfRole"]
```

Comment:
We are _positive_ that this has no adverse effects anywhere else we are using this relationships field? 

Ie in the User's Group list?

Reply:
Just verified this! No problems :)